cmd/shim/main.go

```diff
@@ -424,9 +436,13 @@ func setupBundle() int {
 		}
 	}
 
+	var gpuParams string
 	keepEnv := []string{}
 	for _, env := range spec.Process.Env {
 		switch {
+		case strings.HasPrefix(env, "_DAGGER_GPU_PARAMS"):
```

Comment:
Nit: we shall prefix it as `_EXPERIMENTAL_DAGGER_GPU_PARAMS` for now since it's a new experimental feature.

Reply:
Agree